### DISCUSSION TITLE
Profile flagged, 404 error with profile

### DISCUSSION THREAD

**Discussion by DelibasicIzet** [community/community#113229] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I have access to my account, but it's restricted. Commits are being fetched into repositories but not being deployed, and I can't deploy anything either. Can someone help me? It's been seven days since I opened a request on my primary account, but support isn't responding. I don't know why my account is flagged. Do you have any solutions?

Reinstatement Request #2666054, if support can assist with that.

**Comment by lin20260407** [comment]
my friend. Is there an error image to look at? I don't understand your question.

**Reply by DelibasicIzet** [reply]
This profile from which I initiated the discussion is another profile. My real profile is flagged, and in the settings, it says it's flagged and only I can see it. It's not publicly available, and no one can find it through search or URL. When you enter my profile, you can see that I follow that profile, but it won't display. I've submitted a support ticket for the issue seven days ago, but I haven't received a response. When someone tries to search for me using the URL, it returns a "404 not found" error. Can you please assist me in restoring my account as soon as possible?

**Reply by lin20260407** [reply]
my friend: 
go here https://support.github.com/contact/account
and response your problem

**Reply by DelibasicIzet** [reply]
I opened a request seven days ago at the same place, but they're not responding; I don't know whether to repeat the request or not?

**Comment by akashpawar43** [comment]
I encountered the same issue one month ago, but after filing a complaint, they initially denied my request to unflag my account. However, after some time, my account reappeared in public within four or five days.
you can go here https://support.github.com/contact/account are rise an complaint

**Reply by DelibasicIzet** [reply]
I opened a request seven days ago at the same place, but they're not responding; I don't know whether to repeat the request or not?

**Reply by akashpawar43** [reply]
take backup of all repo, and it depends on for what reason they have flagged, you mostly wait for 1 or 2 weeks.

I hope your account will come back.

**Comment by IzetDelibasic** [comment, marked as answer]
Discussion is closed, the profile has been unflagged and returned to its original state.
